web_external/models/StudyModel.js

```diff
@@ -33,21 +33,28 @@ const StudyModel = Model.extend({
         return new UserCollection(userModels);
     },
 
+    waitingUsers: function () {
+        let userModels;
+        if (this.has('waitingUsers')) {
+            userModels = this.get('waitingUsers').map((user) => {
+                return new UserModel(user);
+            });
+        } else {
+            userModels = [];
```

Comment:
If someone is calling this method and they're not an admin, I'd rather it produce an error / failure, instead of silently returning an empty list.